### DISCUSSION TITLE
Update to German locale [de]

### DISCUSSION THREAD

**Discussion by calipo69** [aclap-dev/video-downloadhelper#879] [Language & Translation]
hello!
[de.txt](https://github.com/user-attachments/files/15856112/de.txt)

I translated the 3 missing strings.
best regards calipo

**Comment by calipo69** [comment]
hello!
I just found a better translation for the following string: 

v9_lic_status_locked2:
**Lizenz gesperrt (überprüfe Deine E-Mails)**
English: License locked (check your email)

hope I could helped!
Calipo69

**Reply by wwilde** [reply]
Maybe a more polite version of it could be considered:

German: "Lizenz ist gesperrt (bitte überprüfe Deine E-Mails)"

What do you think of that suggestion?

**Comment by wwilde** [comment]
And here is another de.txt file:
[de.txt](https://github.com/user-attachments/files/15860263/de.txt)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
